### PR TITLE
Design updates, cleaning up my previous branch

### PR DESCRIPTION
**Pull request by mariamgrig**
<Got rid of the tmp cache files>

I did a bunch of show/view aesthetic changes plus added a new customs css. Added a couple partials for a new header and a footer in the layouts. Also added a static about page. The show view is represented as a pop-up container now. Still need to work on a couple minor changes/css clean up and form design.

### CODE REVIEW

**Comment by Allegraknox** [comment]
Looks good to me!! The only thing I'm not sure about is if the staticPagesController#about needs to be tested for 100% coverage, but since it is just an empty method I would assume it does not need a test and is already 100% covered

**Review by Allegraknox — APPROVED** [review]

File: app/views/items/show.html.erb [hunk 1]
```diff
@@ -1,37 +1,53 @@
...
+        <div class="pop-up-container">
+           <div class="pop-up-container-vertical">
+                <div class="pop-up-wrapper">
+                   <div class="go-back">
+
+                    <%= link_to("Back", items_path) %>
+
+                    </div>
+                    <div class="itemlist">
+                        <div class="itemlist-item-wrapper">
```
Comment (mrubakh) [inline app/views/items/show.html.erb:12]: if you're trying to apply multiple classes at once, I would stay away from creating lots of divs. Instead, you can say `<div class="itemlist-item-wrapper product-details product-left ....etc"`

If that wasn't your intention, let me know.

**Review by mrubakh — APPROVED** [review]
I'll approve this (since I think I'm being picky) but there are 2 things that you may want to change for this commit:

1. the show.html.erb 
2. the db/* and log/* files are still part of the commit

File: app/views/items/show.html.erb (hunk 1 above)
Comment (amycsong98) [reply app/views/items/show.html.erb:12]: +1 on ^

Comment (amycsong98) [reply app/views/items/show.html.erb:12]: you can have a lot of classes for class= classname1 classname2 classname3, etc.

**Review by amycsong98 — APPROVED** [review]
LGTM. maybe update the div classes + then merge?
